ironfish-cli/src/commands/wallet/import.ts

```diff
@@ -71,8 +51,10 @@ export class ImportCommand extends IronfishCommand {
     }
 
     if (account === null) {
-      this.log('No account to import provided')
-      return this.exit(1)
+      CliUx.ux.warn(
+        'Failed to decode the account from the provided input, please continue with the manual input below',
+      )
+      account = await this.manualImport()
```

Comment:
I think this flow got a bit worse in the failure case. If I enter an invalid key, I get asked twice, even though the validation already happens in importTTY. I don't think there's any reasonable way to get to this code path, right?

```
ironfish wallet:import
Paste the output of wallet:export: b4174e1c50453ac1638915136ab9eaff0e7e028c427754ef2163858eb5491d2
 ›   Warning: Failed to decode the account from the provided input, please continue with the manual input below
Enter the account spending key: b4174e1c50453ac1638915136ab9eaff0e7e028c427754ef2163858eb5491d2
Enter a new account name: jason2
InvalidPaymentAddress
```

Reply:
Yea your are right, we should just delete this path and exit if this occurs

Reply:
<img width="1060" alt="Screen Shot 2023-02-07 at 4 22 26 PM" src="https://user-images.githubusercontent.com/26990067/217397111-9f997470-5955-4307-9fbe-bae9a685fe9c.png">

Reply:
<img width="1047" alt="Screen Shot 2023-02-07 at 4 49 21 PM" src="https://user-images.githubusercontent.com/26990067/217400706-57309958-4016-4cd7-8b3f-fa220569e0f0.png">